.github/workflows/build.yml

```diff
@@ -21,7 +22,7 @@ jobs:
       CS_NAMING_KEY:  ${{ secrets.CS_NAMING_KEY }}
       RAILS_VERSION: ${{ matrix.rails-version }}
       ACTIVE_STASH_TEST_COLLECTION_PREFIX: run_${{ github.run_number }}.${{ github.run_attempt }}_job_${{ github.job }}_index_${{ strategy.job-index }}
-      PGDATABASE: activestash_test_run_${{ github.run_number }}.${{ github.run_attempt }}_job_${{ github.job }}_index_${{ strategy.job-index }}
+      PGDATABASE: test
```

Comment:
This was getting truncated for some tests. [Example](https://github.com/cipherstash/activestash/runs/7717469138?check_suite_focus=true#step:4:22):
```
NOTICE:  identifier "activestash_test_run_294.1_job_run-specs-against-ruby-client-main_index_0" will be truncated to "activestash_test_run_294.1_job_run-specs-against-ruby-client-ma"
```

If I'm following correctly, each job sets up a fresh PG, so we don't really need a unique name here. A simple fix is to pick a short, static name.

Reply:
I remember this was put in as part of a fix, will find the PR to see if it gives more context.

Reply:
This is the change https://github.com/cipherstash/activestash/pull/40. I think it might be needed if this job needs to be re run.

Reply:
We still need that fix for `ACTIVE_STASH_TEST_COLLECTION_PREFIX` because we don't want to share collections between jobs (since they're hosted).

I don't think that's an issue for PG since each job gets its own local instance (unless I'm missing something).

I've re-run all jobs for this change a few times and haven't seen a failure yet.